.github/ISSUE_TEMPLATE/story.md

```diff
@@ -37,10 +37,11 @@ What else should contributors [keep in mind](https://fleetdm.com/handbook/compan
 - [ ] REST API changes: TODO <!-- Specify changes as a draft PR to the REST API doc page and request the API DRI for review (codeowner reviews are not automatically requested for drafts). Remove this checkbox if there are no changes necessary. Move this item to the engineering list below if engineering will design the API changes. -->
 - [ ] Fleet's agent (fleetd) changes: TODO <!-- Specify changes to fleetd. If the change requires a new Fleet (server) version, consider specifying to only enable this change in new Fleet versions. Remove this checkbox if there are no changes necessary. -->
 - [ ] Permissions changes: TODO <!-- Specify changes as a draft PR to the Manage access doc page. If doc changes aren't necessary, explicitly mention no changes to the doc page. Remove this checkbox if there are no permissions changes. -->
-- [ ] Outdated documentation changes: TODO <!-- Specify required documentation changes (public-facing fleetdm.com/docs or contributors) & redirects to add to /website/config/routes.js. -->
 - [ ] Changes to paid features or tiers: TODO  <!-- Specify "Fleet Free" or "Fleet Premium".  If only certain parts of the user story involve paid features, specify which parts.  Implementation of paid features should live in the `ee/` directory. -->
 
 ### Engineering
+- [ ] Reference documentation changes: TODO <!-- Specify references documentation changes at fleetdm.com/docs -->
+- [ ] Usage documentation changes: TODO <!-- Specify usage documentation changes at fleetdm.com/docs/using-fleet -->
```

Comment:
This brings up the question of who the CODEOWNER of the docs/contributing should be since everything in /docs still requires a design review. Should that be changed to move faster?

Reply:
at least for `contrib` I mean

Reply:
@georgekarrv We're going to keep @rachaelshaw was the docs DRI to ensure consistency across all the different folks contributing. @eashaw will be her backup when Rachael is out.

Reply:
@georgekarrv Contributing docs could be me and EMs, I think. What do you think @rachaelshaw?

Reply:
@lukeheath that makes sense to me; I think I was codeowner of the contrib doc because we assumed a lot of that stuff might get moved into the public-facing docs at some point, but that hasn't really been the case. (It's more often been the other way around.) I think it makes sense to always re-review anything that gets moved into the regular docs anyway 👍

Reply:
I agree with @georgekarrv that we will eventually need to train engineers to review and approve docs.
As an immediate term I'll ask my team to separate the docs PRs to avoid bottlenecks.